spec/14-classes.md

```diff
@@ -1994,4 +2064,22 @@ base class. An instance of this type is automatically created when a
 non-object is [converted to an object](08-conversions.md#converting-to-object-type), or the [member selection
 operator](10-expressions.md#member-selection-operator) is applied to `NULL`, `FALSE`, or an empty string.
 
+###Class `TypeError`
+
+An instance of this class is thrown when any of the following occurs in [strict mode](13-functions.md#type-check-modes):
```

Comment:
The first two bullets also apply to weak mode (coercion semantics notwithstanding).